mos-platform/rp6502/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+platform(rp6502 COMPLETE HOSTED PARENT common)
+
+if(NOT CMAKE_CROSSCOMPILING)
+  return()
+endif()
+
+install(FILES
+  rp6502.h
+TYPE INCLUDE)
+install(FILES link.ld TYPE LIB)
+
+add_platform_object_file(rp6502-crt0-o crt0.o
+  init-cpu.s
+)
+
+add_platform_library(rp6502-crt0
+  exit.c
+)
+merge_libraries(rp6502-crt0
+  common-crt0
+  common-init-stack
+  common-copy-zp-data
+  common-zero-bss
+  common-exit-loop
```

Comment:
This provides an implementation of `_Exit` that loops, but you've already provided one, so this isn't needed. Instead, include `exit-custom` to pull in an `exit()` implementation that calls your `_Exit()`.

Reply:
done.